Files/Constants.cs

```diff
@@ -2,6 +2,27 @@
 {
     public static class Constants
     {
+        public static class ImageRes
+        {
+            // See imageres.dll for more icon indexes to add
+            public const int QuickAccess = 1024;
+            public const int Desktop = 183;
```

Comment:
Does this icon need to be hardcoded since we can just detect it from the path?

Reply:
There are extra constant values we have access to that we don't fetch icons for. I figured we can keep them in case we need them for something else. But many of them aren't selected to be extracted from the DLL.